packages/jest-worker/src/workers/ChildProcessWorker.ts

```diff
@@ -147,6 +153,13 @@ export default class ChildProcessWorker implements WorkerInterface {
     }
   }
 
+  monitorHeartbeat(onExceeded: () => any): void {
+    clearTimeout(this._heartbeatTimeout);
+    this._heartbeatTimeout = setTimeout(() => {
+      onExceeded();
+    }, this._options.workerHeartbeatTimeout);
```

Comment:
```suggestion
    }, this._options.workerHeartbeatTimeout).unref();
```

I think - we don't want this to keep the process open or some such